qiskit_optimization/problems/quadratic_program.py

```diff
@@ -1025,6 +1144,40 @@ def from_docplex(self, model: Model) -> None:
             else:
                 raise QiskitOptimizationError("Unsupported constraint sense: {}".format(constraint))
 
+        # get indicator constraints
+        for constraint in model.iter_logical_constraints():
```

Comment:
Isn't it `iter_indicator_constraints`? https://ibmdecisionoptimization.github.io/docplex-doc/mp/docplex.mp.model.html?highlight=iter_#docplex.mp.model.Model.iter_indicator_constraints

Reply:
`iter_logical_constraints()` works since it's the super class of `IndicatorConstraint`, but I think `iter_indicator_constraint` is more specific and more appropriate. I fixed it.